src/pages/MinerDashboard/Header/MinerDetails.tsx

```diff
@@ -116,11 +117,22 @@ export const MinerDetails: React.FC<{
           <div>{settings && coin ? payoutLimit : <Skeleton width={40} />}</div>
         </Item>
         {
-        (maxFeePrice !== undefined && maxFeePrice > 0) ? (
+        (maxFeePrice !== undefined) ? (
         <Item>
           <div>Gas Limit:&nbsp;</div>
-          <div>{maxFeePrice}{' '}{feeDetails?.unit}</div>
-        </Item>) : (null)
+          {maxFeePrice && feeDetails ?  <div>{maxFeePrice + " " + feeDetails.unit}</div> :
+          <div>None Set&nbsp;&nbsp;
+            {/*}<Tooltip>
+              <TooltipContent
+                message={'No gas limit has been set. The payment transaction will be executed when the payment limit is reached, regardless of current gas fees.'}
+              />
+            </Tooltip>*/}
+          </div>}
+        </Item>) : (
+        <Item>
+          <div>Gas Limit:&nbsp;</div>
```

Comment:
First, this should be "Gas Price Limit". Gas Limit is a completely different thing.

Also, this value name should change depending on the coin selected. With Ethereum it is Gas Price, with Bitcoin it is sat/b, etc. This is why the generic `maxFeePrice` name was chosen in the API. cc @march08